app/views/job_invocations/_form.html.erb

```diff
@@ -75,10 +75,59 @@
     <legend><%= _('Schedule Job') %></legend>
     <%= radio_button_f f, :trigger_mode, :value => 'immediate', :text => _("Run Job Now"), :class => 'trigger_mode_selector' %>
     <%= radio_button_f f, :trigger_mode, :value => 'future', :text => _("Schedule Future Job"), :class => 'trigger_mode_selector' %>
+    <%= radio_button_f f, :trigger_mode, :value => 'recurring', :text => _("Set up recurring Job"), :class => 'trigger_mode_selector' %>
     <fieldset id="trigger_mode_future" class="trigger_mode_form <%= 'hidden' if @composer.job_invocation.trigger_mode != :future %>">
       <%= text_f f, :start_at, :label => _('Start at') %>
       <%= text_f f, :start_before, :label => _('Start before') %>
     </fieldset>
+    <fieldset id="trigger_mode_recurring" class="trigger_mode_form <%= 'hidden' if @composer.job_invocation.trigger_mode != :recurring %>">
+      <%= selectable_f f, :input_type, %w(cronline monthly weekly daily hourly), {}, :label => _("Repeats") %>
+      <%= f.fields_for 'recurring_options' do |recurring_options_fields| %>
```

Comment:
A potential helper that  might go info foreman-tasks for the field-set for recurring logic?